<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomListWidgetItem</class>
 <widget class="QWidget" name="CustomListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lb_image">
     <property name="text">
      <string>image</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,20,5">
       <item>
        <widget class="QLabel" name="lb_newMsgNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_friendName">
         <property name="text">
          <string>userName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_lastTime">
         <property name="text">
          <string>lastTime</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,2">
       <item>
        <widget class="QLabel" name="lb_lastMsg">
         <property name="text">
          <string>lastMsg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_voice">
         <property name="text">
          <string>voice</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
